templates/cisco_asa_show_version.template

```diff
@@ -13,7 +13,7 @@ Value MAX_INTF (\d+)
 Value MAX_VLANS (\d+)
 Value FAILOVER (\S+)
 Value CLUSTER (\S+)
-Value SERIAL (\S+)
+Value List SERIAL (\S+)
 Value LAST_MOD (.+)
 
```

Comment:
I see some issues with the template regarding your newer version and I think this time will be great to update those so it captures everything properly.

Could you update https://github.com/networktocode/ntc-templates/blob/565539f5993a7bd35ea4d4471ad5f0b6c226dd1b/templates/cisco_asa_show_version.template#L9 to be `(\d+|\S+)`.This will help capture previous code version of number of interfaces (numeral) and what your newer version is (word).

Also, https://github.com/networktocode/ntc-templates/blob/565539f5993a7bd35ea4d4471ad5f0b6c226dd1b/templates/cisco_asa_show_version.template#L27 could you update it to be `^\s*\d+:.\S+\s*${INTERFACES}.*`. It stops capturing interfaces at 10+ due to there no longer being a space. The change is **\s*** captures 0 or more spaces which should capture previous and new output.